subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -263,7 +264,12 @@ abstract class AbstractSmokeTest extends Specification {
             .withProjectDir(testProjectDir)
             .forwardOutput()
             .withArguments(
-                tasks.toList() + outputParameters() + repoMirrorParameters() + configurationCacheParameters()
+                tasks.toList() + outputParameters() + repoMirrorParameters() + configurationCacheParameters() +
+                [
```

Comment:
Looks like you need to make sure this is actually a list of String: https://ge.gradle.org/s/cajgqqth6vk34/tests/:smoke-test:gradleBuildSmokeTest/org.gradle.smoketests.GradleBuildBinaryDistributionConfigurationCacheSmokeTest/can%20build%20and%20install%20Gradle%20binary%20distribution%20with%20configuration%20cache%20enabled?focused-exception-line=0-0&top-execution=1

`GString` doesn't work out of the box. Check the other methods.